Dockerfile

```diff
@@ -5,6 +5,8 @@ MAINTAINER Timescale https://www.timescale.com
 
 ENV TIMESCALEDB_VERSION 0.10.0
 
+COPY docker-entrypoint-initdb.d/* /docker-entrypoint-initdb.d/
```

Comment:
I'd recommend avoiding wildcard copying to ensure consistent builds. I am thinking of the case where you've accidentally left something in your local `/docker-entrypoint-initdb.d/`, which then potentially gets included in the official release Docker image.

Probably better to have an explicit copy of the file you'd like to include.